ubuntu/ubuntu.go

```diff
@@ -65,15 +73,25 @@ type Status struct {
 }
 
 func Update() error {
+	var err error
 	gc := git.Config{}
 	dir := filepath.Join(utils.CacheDir(), "ubuntu-cve-tracker")
-	if _, err := gc.CloneOrPull(repoURL, dir, "master", false); err != nil {
+	for _, url := range repoURLs {
+		_, err = gc.CloneOrPull(url, dir, "master", false)
+		if err == nil {
+			break
+		} else {
+			log.Printf("failed to clone or pull: %s: %v", url, err)
+		}
```

Comment:
[nit]

```suggestion
		if err != nil {
			log.Printf("failed to clone or pull: %s: %v", url, err)
			continue
		}
		break
```

Reply:
aha, I thought of writing it the way you suggests but changed it to the current one :facepalm: